assets/translations/en-us.yaml

```diff
@@ -150,6 +154,22 @@ suffix:
 ##############################
 # Components & Pages
 ##############################
+about:
+  title: About
+  versions:
+    title: Versions
+    component: Component
+    version: Version
+    cli: CLI
+    helm: Helm
+    machine: Machine
+    releaseNotes: 'Release notes can be found <a href="/docs/release-notes">here</a>.'
```

Comment:
This needs to be done as a nuxt link - this causes a full UI reload when clicked as is.

Reply:
Done